shared/src/business/useCases/docketEntry/fileCourtIssuedDocketEntryInteractor.test.js

```diff
@@ -302,4 +302,32 @@ describe('fileCourtIssuedDocketEntryInteractor', () => {
       secondaryDate: '2019-03-01T21:40:46.415Z',
     });
   });
+
+  it('should set isDraft to false on a document when creating a court issued docket entry', async () => {
+    await fileCourtIssuedDocketEntryInteractor({
+      applicationContext,
+      documentMeta: {
+        caseId: caseRecord.caseId,
+        date: '2019-03-01T21:40:46.415Z',
+        documentId: '7f61161c-ede8-43ba-8fab-69e15d057012',
```

Comment:
Just wondering if there should be a mock constant here much like you appear to be transitioning to below with cases:
https://github.com/ustaxcourt/ef-cms/pull/310/files#diff-b6185c4c4afa6da5508a1aa87d8dd80aL83

```
-        caseId: 'c54ba5a9-b37b-479d-9201-067ec6e335bb',
+        docketNumber: MOCK_CASE.docketNumber,
```

Reply:
Hmm, looks like there might already be a related Trello ticket? #647

Reply:
Yes, we're continuing to move to the mock case constants to make changes to the case object easier to test.